packages/components/src/Button/button.stories.tsx

```diff
@@ -9,7 +9,7 @@ import {
   getAllRecommendedVariants,
   getAllVariantsWithStates,
   getLargeVariantsOnDarkBackgroundWithStates,
-} from "../storyUtils/clickableStyleUtils.stories";
+} from "../storyUtils/clickableStyleUtils";
```

Comment:
Removed `.stories` from the filename. Otherwise the linter thinks it has stories in it and tries to apply its rules there.

Reply:
Oh, I see. I think `.stories` was there for Tailwind

Reply:
Updated our Tailwind config to look at anything under storyUtils/

Reply:
makes sense 👍